docker-compose-services/redis/docker-compose.redis.yaml

```
@@ -5,7 +5,7 @@ version: "3.6"
 services:
   redis:
     container_name: ddev-${DDEV_SITENAME}-redis
-    image: redis:4
+    image: redis:latest
```

Using "latest" is always a bad idea with docker images.  I think looking at https://hub.docker.com/_/redis?tab=description&page=1&ordering=last_updated what you want is `redis: 6.2`
```suggestion
    image: redis:6.2
```

---

I can confirm it as working using redis:6.2